zuul.d/projects.yaml

```diff
@@ -65,3 +65,4 @@
     templates:
```

Comment:
ok and now i remember you have a make target that will generate this file so i shouldn't update it directly... 

k updating.